plugins/woocommerce-blocks/assets/js/blocks/coming-soon/edit.tsx

```diff
@@ -118,16 +115,23 @@ export default function Edit( { attributes, setAttributes } ) {
                     margin: 0;
                 }
                 .woocommerce-coming-soon-banner-container {
-                    padding-inline: min(6.5rem, 8vw);
+                    padding-inline: min(5.5rem, 8vw);
                     margin: 0;
+                    height: 100%;
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: space-between;
+                }
+                .woocommerce-coming-soon-banner-container > .wp-block-group__inner-container {
```

Comment:
This is required for classic themes, `wp-block-group__inner-container` appears out of nowhere 🤷